src/containers/Inspector.tsx

```diff
@@ -134,10 +124,11 @@ const Inspector: React.FC<IProps> = (props) => {
         name: props.request ? props.request.method : "New Tab",
         content: props.request || { ...emptyJSONRPC },
         url: props.url || "",
+        openrpcDocument: props.openrpcDocument,
       },
     ],
   );
-  const [id, incrementId] = useCounter(0);
+  const id = 0;
```

Comment:
is this problematic that the id will always be 0?

Reply:
i think incrementing id's and keeping track is a bad path so removing it is fine

Reply:
Offline conversation. Rationale was that in general this should be fine for inspectors usecase. The escape clause is that you can always in the inspector editor set the id field yourself. 

The question mark here was for a long standing request to a service, and then making a request with the same id, if that would cause shenanigans. That said inspector doing the incrementing is a bit of an overreach.